frontend/src/components/maps/leaflet/InfoSlideOut/AssociatedParcelsList.tsx

```diff
@@ -0,0 +1,51 @@
+import { IBuilding } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+
+interface IAssociatedParcels {
+  /** the selected property information */
+  propertyInfo: IBuilding | null;
```

Comment:
would it make more sense that IAssociatedParcels just contained a list of IParcel?

Reply:
oh that does make a lot more sense, I'll change that